smk/binning_preparation.smk

```diff
@@ -20,11 +20,6 @@ use rule binning_preparation_base, binning_preparation_avamb from print_versions
 
 snakefile_name = print_versions.get_smk_filename()
 
-localrules: check_depth_batches, combine_contigs_depth_batches, combine_fasta_batches, \
-            combine_fasta, check_depths, combine_depth, config_yml_binning, \
-            make_abundance_npz
-
```

Comment:
If make_abundance_npz is no longer a localrule, it should have `resources.mem` specified, just in case people don't run snakemake with default resources.

Reply:
Ok, now you are forcing me to run a titration experiment to estimate memory requirement... I will work on it...

Reply:
Now added mem estimation (a37cb43)